runtime/common/src/lib.rs

```diff
@@ -74,10 +68,7 @@ macro_rules! decl_common_types {
             frame_system::ChainContext<Runtime>,
             Runtime,
             AllPalletsWithSystem,
-            (
-                zrml_prediction_markets::migrations::UpdateMarketsForDeadlines<Runtime>,
-                zrml_prediction_markets::migrations::MigrateMarketIdsPerBlockStorage<Runtime>,
-            )
```

Comment:
> Ok, now I ask 😁: Are these migrations already executed? Migrations get executed when a runtime upgrade happens. So with every release cycle. So Harald makes the `0.3.6` release from [this commit](https://github.com/zeitgeistpm/zeitgeist/commit/8c6395fc152f04b49e1e6866ae0bbfbb8df5dfd3) (which includes the deleted migrations here), right?
> 
> And this PR is part of `0.3.7`. So we collect all coming storage migrations into this `frame_executive` for `0.3.7`, okay.

Correct.

> Another question: The second `type Executive` is when the feature parachain is not activated. So, it is required for each migrations (which has nothing to do with the parachain stuff like pallet_parachain_staking) to pass it in too? So this is the standalone mode, in case Zeitgeist looses the parachain slot? So it's better for each migration to execute it for the standalone mode too. But, when Zeitgeist always would have a parachain slot, this would be not necessary?

First of all, nothing speaks against making sure that the migrations execute on the standalone version, as well. If a migration for, say, `market-commons` was only executed on parachains, I would be very confused. We should also try to treat the development of Zeitgeist as if other developers are using our code to run chains, so executing the upgrade on standalone networks is often necessary.